app/scripts/migrations/103.test.ts

```diff
@@ -0,0 +1,256 @@
+          internalAccounts: {
+            accounts: {
+              [expectedUUID]: expectedInternalAccount(
+                MOCK_ADDRESS,
+                `Account 1`,
+              ),
+            },
+            selectedAccount: expectedUUID,
+          },
+        },
+        PreferencesController: expect.any(Object),
+      });
+    });
+
+    it('should keep the same name from the identities', async () => {
+      const oldData = createMockState(
+        createMockPreferenceControllerState([
+          { name: 'a random name', address: MOCK_ADDRESS },
+        ]),
+      );
+      const oldStorage = {
+        meta: { version: 102 },
+        data: oldData,
+      };
+      const newStorage = await migrate(oldStorage);
+      expect(newStorage.data).toStrictEqual({
+        PreferencesController: expect.any(Object),
+        AccountsController: {
+          internalAccounts: {
+            accounts: {
+              [expectedUUID]: expectedInternalAccount(
+                MOCK_ADDRESS,
+                `a random name`,
+              ),
+            },
+            selectedAccount: expectedUUID,
+          },
+        },
+      });
+    });
+
+    it('should be able to handle multiple identities', async () => {
+      const oldData = createMockState({
+        identities: {
+          [MOCK_ADDRESS]: { name: 'Account 1', address: MOCK_ADDRESS },
+          [MOCK_ADDRESS_2]: { name: 'Account 2', address: MOCK_ADDRESS_2 },
+        },
+        selectedAddress: MOCK_ADDRESS,
+      });
+
+      const oldStorage = {
+        meta: { version: 102 },
+        data: oldData,
+      };
+
+      const newStorage = await migrate(oldStorage);
+
+      expect(newStorage.data).toStrictEqual({
+        AccountsController: {
+          internalAccounts: {
+            accounts: {
+              [expectedUUID]: expectedInternalAccount(
+                MOCK_ADDRESS,
+                `Account 1`,
+              ),
+              [expectedUUID2]: expectedInternalAccount(
+                MOCK_ADDRESS_2,
+                `Account 2`,
+              ),
+            },
+            selectedAccount: expectedUUID,
+          },
+        },
+        PreferencesController: expect.any(Object),
+      });
+    });
+  });
+
+  describe('createSelectedAccountForAccountsController', () => {
+    it('should select the same account as the selected address', async () => {
+      const oldData = createMockState();
+      const oldStorage = {
+        meta: { version: 102 },
+        data: oldData,
+      };
+      const newStorage = await migrate(oldStorage);
+      expect(newStorage.data).toStrictEqual({
+        PreferencesController: expect.any(Object),
+        AccountsController: {
+          internalAccounts: {
+            accounts: expect.any(Object),
+            selectedAccount: addressToUUID(MOCK_ADDRESS),
+          },
+        },
+      });
+    });
+
+    it("should leave selectedAccount as empty is there aren't any selectedAddress", async () => {
```

Comment:
The body of this test is identical to the previous one

Reply:
Fixed on 1b553df2b5ddab5f9b1afa73e4f140c06633794a